tests/docs.rs

```diff
@@ -0,0 +1,10 @@
+extern crate docmatic;
+
+#[test]
+fn readme_test() {
+    let readme = std::path::Path::new(file!()).canonicalize().unwrap();
+    let readme = readme.parent().unwrap().parent().unwrap().join("README.md");
+    let readme = readme.to_str().unwrap();
+
+    docmatic::assert_file(readme);
```

Comment:
Why? The readme has no runnable tests

Reply:
The readme does have this:

```rust,no_run
extern crate globwalk;

use std::fs;

for img in globwalk::glob("*.{png,jpg,gif}").unwrap() {
    if let Ok(img) = img {
        fs::remove_file(img.path()).unwrap();
    }
}
```

Even if that isn't runnable, docmatic will ensure it compiles.

Reply:
I see, very well.